README.md

```diff
@@ -114,147 +80,63 @@ pub fn render() {
     App::builder(update, view).build_and_start();
 }
 ```
-
-## Building and running
-
-To build your app, run `cargo make build`, and to host on a dev server, run `cargo make serve`.
-
-For a more robust starting setup, check out Martin Kavik's [seed-quickstart-webpack repo](https://github.com/seed-rs/seed-quickstart-webpack).
-
-## Running included examples
-
-To run an example located in the [examples folder](https://github.com/seed-rs/seed/tree/master/examples),
-run `cargo make start example_name`, where you replace `example_name` with the example name. Eg:
-`cargo make start counter`.
-
-Some examples also require to run API server in another terminal window - `cargo make start_server example_name`.
-
-When server(s) are running, open [127.0.0.1:8000](http://127.0.0.1:8000) in your browser.
+- Run other [examples](examples/) with `cargo make start example_name`.
+- Check out [todomvc](examples/todomvc) if you're feeling confident!
+- Also see [webpack quickstart](https://github.com/seed-rs/seed-quickstart-webpack).
+
+# Why Use Seed
+
+### Rust/WASM
+You may prefer writing in Rust and appreciate its benefits, including:
+- Rust **safety**.
+- Rust **compile-time error, type, and immutability checking**.
+- Rust built-in testing.
+- Rust is fast.
+- Cargo packages.
+
+### Consistency
+- The API is flexible, and avoids OOP boilerplate. Its integrated routing and message system avoids the dependency glue-code associated with Redux and React-Router.
+- No distinction between logic and display code.
+- No distinction between components and normal functions.
+- No restrictions on comments.
+
+### Performance
+- Seed uses `wasm-bindgen`'s zero-cost abstractions and `web-sys` internally, meaning Seed _will be_ optimized for performance.
+
+### Development
+- Very active development.
+- Macro syntax removes the need for transpiling and integrates naturally and flexibly with the language.
+- Seed has a _batteries-included_ approach.
+- Built in elm-like architecture, no need for another state manager.
+- Maybe you don't want to code business logic in a purely-functional language.
+
+See more on our [about](https://seed-rs.org/guide/about) page.
+
+# Why Not Use Seed
+- Seed is under rapid development, and some breaking changes are likely. However, Seed is more than stable enough for personal projects.
```

Comment:
I'm writing production apps as a freelancer.

Reply:
I felt like the [why not use seed](https://github.com/mikeyoz/seed#why-not-to-use-this-and-stick-with-js) section of the original readme was a great idea but maybe a bit deprecating. I tried to make the language lighter, but we can edit this further. Do you have a suggestion?

Reply:
- Just not to scare companies - e.g. `However, Seed is more than stable enough for personal projects and production apps are in development.`